### PR TITLE
Adding support for sending Audit Logs from Connector   

### PR DESCRIPTION
**Pull request by koala7659**
Following code modification have been made:

1. Adding tenantMappingService (Director hydrator) and claims map into connector oathkeeper rules to enrich Connector requests
2. Using Gateway for sending Audit Logs for connector requests
3. Modification of connector upstream value for connectivity-adapter

### CODE REVIEW

File: components/connectivity-adapter/internal/connectorservice/api/certificates.go [hunk 1]
```diff
@@ -48,10 +48,8 @@ func (ch *certificatesHandler) SignCSR(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	contextLogger.Info("Generating certificate")
```
Comment (akgalwas) [inline components/connectivity-adapter/internal/connectorservice/api/certificates.go:51]: I think we should not remove this log entry.

File: components/connectivity-adapter/internal/connectorservice/connector/client.go [hunk 2]
```diff
@@ -18,15 +17,11 @@ type client struct {
 	timeout       time.Duration
 }
 
-func NewClient(compassConnectorAPIURL string, timeout time.Duration) (Client, error) {
-	gqlExternalAPIClient := graphql.NewClient(compassConnectorAPIURL, graphql.WithHTTPClient(&http.Client{}))
-
-	client := &client{
-		gqlAPIClient: gqlExternalAPIClient,
-		timeout:      timeout,
+func NewClient(gqlClient *graphql.Client, timeout time.Duration) Client {
```
Comment (akgalwas) [inline components/connectivity-adapter/internal/connectorservice/connector/client.go:20]: timeout argument is no longer used (timeout value hardcoded)

File: components/connectivity-adapter/internal/connectorservice/api/certificates.go (hunk 1 above)
Comment (koala7659) [reply components/connectivity-adapter/internal/connectorservice/api/certificates.go:51]: Ok, fixed. My mistake

File: components/connectivity-adapter/internal/connectorservice/connector/client.go (hunk 2 above)
Comment (koala7659) [reply components/connectivity-adapter/internal/connectorservice/connector/client.go:20]: That's correct

**Pull request merged**